bin/renew-token

```diff
@@ -0,0 +1,75 @@
+#!/bin/env python
+
+import logging
+import os
+import random
+import requests
+import string
+import sys
+
+from argparse import ArgumentParser
+from firkinize.configstore.consul import Consul
+from vaultlib.ca import VaultCA
+
+logging.basicConfig(level=logging.DEBUG)
+
+LOG = logging.getLogger(__name__)
+
+# Utility for creating new vouch token
+def parse_args():
+    parser = ArgumentParser(description='Initialize vouch signing service')
+    parser.add_argument('--config-url', default='http://localhost:8500',
+                        help='Address of the config node, default http://localhost:8500')
+    parser.add_argument('--customer-id',
+                        help='The keystone customer id', required=True)
+    parser.add_argument('--region-id',
+                        help='The region id for which to bootstrap the keystone endpoint',
+                        required=True)
+    parser.add_argument('--config-token',
+                        help='config access token, also looks for '
+                             'env[\'CONSUL_HTTP_TOKEN\']')
+    return parser.parse_args()
+
+
+def get_vault_admin_client(consul, customer_uuid):
```

Comment:
looks like some functions, such as get_vault_admin_client,  create_host_signing_role, and create_host_signing_token  have been copied from the init-region? There is a possibility of drift if someone modifies these functions (for e.g, get_vault_admin_client  in init-region but not in renew-token). maybe you could try importing these functions instead? I guess importing functions is not allowed if the Python file name contains a hyphen (in this case, init-region contains a hyphen)
maybe you can move these common functions to new `common.py` and import these common functions both in init-region and renew-token.

Reply:
Also, I liked the @scriptory's suggestion https://platform9.atlassian.net/browse/INF-1648?focusedCommentId=89171, 
Call the same function/script both in init-region and cronjob to issue token. currently, you still have to call below functions both in renew-token & init-region. maybe create a common wrapper function for the token issue and call it in both scripts.
```
    vault = get_vault_admin_client(consul, args.customer_id)
    rolename = create_host_signing_role(vault, consul, args.customer_id)
    create_host_signing_token(vault, consul, args.customer_id, rolename)
```

Reply:
New TC Build Run with changes: https://teamcity.platform9.horse/buildConfiguration/Pf9project_ContainerBuilds_VouchContainer/2864082

Created a `common.py` script with original init-region code in it. And created 2 separate `init-region` and `token-renew` scripts that call common.py to ensure drift issues as mentioned in review comments are avoided.

```
# kubectl exec -it vouch-keystone-7789dd5669-5wcwb -n sanchit-vouch-tp -c vouch-keystone -- bash
...
bash-5.1# ls -lrt
total 7236
-rwxr-xr-x    1 root     root       7408544 Jun 29  2017 consul-template
lrwxrwxrwx    1 root     root            26 Sep 25 21:52 renew-token -> /usr/local/bin/renew-token
lrwxrwxrwx    1 root     root            26 Sep 25 21:52 init-region -> /usr/local/bin/init-region
lrwxrwxrwx    1 root     root            24 Sep 25 21:52 common.py -> /usr/local/bin/common.py
```